go/sts/TakeRole/TakeRole.go

```diff
@@ -0,0 +1,68 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+// snippet-start:[sts.go.take_role]
+package main
+
+// snippet-start:[sts.go.take_role.imports]
+import (
+    "flag"
+    "fmt"
+
+    "github.com/aws/aws-sdk-go/aws/session"
+    "github.com/aws/aws-sdk-go/service/sts"
+    "github.com/aws/aws-sdk-go/service/sts/stsiface"
+)
+// snippet-end:[sts.go.take_role.imports]
+
+// TakeRole gets temporary security credentials to access resources
+// Inputs:
+//     svc is an Amazon STS service client
+//     roleARN is the ARN of the role to assume
```

Comment:
...the **Amazon Resource Name (ARN)**...

Reply:
Fixed